docs/bokeh/source/docs/first_steps/examples/first_steps_5_vectorize_color.py

```diff
@@ -7,7 +7,7 @@
 y = random.sample(range(0, 100), 26)
 
 # generate list of rgb hex colors in relation to y
-colors = ["#%02x%02x%02x" % (255, int(round(value * 255 / 100)), 255) for value in y]
+colors = [f"#{255:02x}{int((value * 255) / 100):02x}{255:02x}" for value in y]
```

Comment:
Slightly off-topic, but if are aiming to make this as simple as possible for users, as it is quite early in the docs, we can keep the colors as RGB tuples and avoid converting to strings:
```suggestion
colors = [(255, int(round(value * 255 / 100)), 255) for value in y]
```

Reply:
I was thinking as I updated these that I didn't really like this code at all, and then I also saw another PR use numpy arrays or colors which seemed better. I definitely agree all the string color examples could be improved. I'll make a separate issue so that can happen in its own smaller PR.

Reply:
https://github.com/bokeh/bokeh/issues/12760